setup.py

```diff
@@ -68,6 +69,18 @@ def pkgconfig(*packages, **kw):
     return kw
 
 
+def copy_files(src, dst):
```

Comment:
@tito the reason is that copytree insists on the target directories must not exist, but here it definitely exists. There are similar issues if the subdirectories already exist. the copy_files method doesn't mind existing directories.